<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal Budgetting System for Students</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../logo_plain/blob.png</normaloff>../../logo_plain/blob.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>185</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newWalButton">
       <property name="text">
        <string>Create new wallet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newAccButton">
       <property name="text">
        <string>Create new account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="creditButton">
       <property name="text">
        <string>Credit / debit amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewSheetButton">
       <property name="text">
        <string>View spreadsheet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="text">
        <string>About PBS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>21</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <action name="actionWallet">
   <property name="text">
    <string>Wallet</string>
   </property>
  </action>
  <action name="actionBank_Account">
   <property name="text">
    <string>Bank Account</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionSpreadsheet">
   <property name="text">
    <string>Spreadsheet</string>
   </property>
  </action>
  <action name="actionData_entry">
   <property name="text">
    <string>Data entry</string>
   </property>
  </action>
  <action name="actionAbout_PBS">
   <property name="text">
    <string>About PBS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
